.gitignore

```diff
@@ -1,131 +1,23 @@
-*.suo
-*.user
-.DS_Store
-
-# Logs
-logs
-*.log
-
-# Runtime data
-pids
-*.pid
-*.seed
-
-# Dependency directory
-# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
-node_modules
-npm-debug.log
```

Comment:
I'd say it was a mistake to have it there, because we shouldn't be checking in code if npm doesn't cleanly run.